features/support/paths.rb

```diff
@@ -609,8 +611,11 @@ def path_to(page_name, *args)
     when /^the integration show page for (service ".+?")/
       admin_service_integration_path(Transform $1)
     when /^the integration page for (service ".+?")/
+      # TODO: THREESCALE-3759 edit page no longer exist, remove and place for show
```

Comment:
Just remove it in https://github.com/3scale/porta/pull/2288#discussion_r537506820 and do not replace it for anything. It just shouldn't be accessible and shouldn't exist. If there is any test pointing to it, we have to remove it or fix it depending on the test.

Reply:
Yep, that's what the comment meant